README.md

```diff
@@ -1,6 +1,55 @@
-# Perfeed
-Personalized, AI assistant that runs with your engineering teams
+# 🚀 Perfeed
+A personalized AI assistant tailored to streamline engineering team workflows.
```

Comment:
Let's make it clear that its main and only function is to summarize PRs to give a weekly summary. More to come later based on feature requests.

Reply:
add the feature section below